lego/apps/events/models.py

```diff
@@ -532,7 +565,9 @@ def add_to_waiting_list(self, user):
             },
         )[0]
 
-    def pop_from_waiting_list(self, to_pool: Pool | None = None) -> Registration | None:
+    def pop_from_waiting_list(
+        self, to_pool: Optional[Pool] = None
+    ) -> Registration | None:
```

Comment:
Not sure if I like `model | None` or `Optional[model]` best. All functions returning something optional uses the pipe, while variables and args use `Optional`. Let me know what you think.

Reply:
I think it's better to stick to one thing. And I'm leaning towards `Pool | None` just because it's a bit more verbose